db-ops/job-create-db-postgres.yml.tpl

```diff
@@ -39,9 +32,14 @@ spec:
         image: $DOCKER_IMAGE_TAG
         command:
         - sh
-        args:
-        - '-c'
-        - psql -U$MASTER_RDS_USERNAME -h $DB_ENDPOINT postgres -v ON_ERROR_STOP=1 -a -f/db-ops/create-user.sql > /dev/null
+        - -c
+        - |
+          for db in $DB_LIST; do
+            [[ echo '\l' | psql -U$MASTER_RDS_USERNAME -h $DB_ENDPOINT postgres | grep \$db ]] || \
+            cat /db-ops/create-user.sql | \
+            sed \"s/__DB_NAME/\$db/g\" | \
```

Comment:
Ah I see, you're using __ as a placeholder. How about we just stick with the same behaviour as in `create-db-user` and do proper variable substitution using `eval "echo \"$(cat /db-ops/create-user.sql)\"`?

Reply:
that __DB_NAME (now {{ DB_NAME }} can not be decided before the loop in Job.